Shared/Extensions/StringExtensions.swift

```diff
@@ -178,4 +178,21 @@ extension String {
             return nil
         }
     }
+    
+    /// Encode Strings which are not sanitized for displaying
+    /// - Returns: Encoded String
+    public func encodeContaminatedString() -> String {
```

Comment:
looks like a simple function, but maybe add a small unit test for it?
this can also be a computer property instead of a `func`

Reply:
can we use this code or remove it, looks like we are duplicating code here: https://github.com/brave/brave-ios/blob/development/Shared/Extensions/WKWebViewExtensions.swift#L83?

Reply:
Yeap I am carrying that exact function inside StringExtensions

Reply:
Duplicate method is removed and added as computed property and unit tests are added


Changes at https://github.com/brave/brave-ios/pull/4076/commits/26c16ac99d70d8a804ceba3e5e6970746bac714e